runtime/python/pose_estimation/pose_estimation.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+import multiprocessing as mp
+from multiprocessing import Process
+
+from pathlib import Path
+from zenlog import logging
+from PIL import Image
+
+
+from pose_estimation_utils import *
```

Comment:
write all the function names so it's clear where they are from

Reply:
`from pose_estimation_utils import output_data_type2dict, preprocess, postprocess_and_visualize
`